.solr_wrapper.yml

```diff
@@ -3,3 +3,4 @@
 collection:
   dir: solr/conf/
   name: blacklight-core
+version: 6.4.0
```

Comment:
This wasn't intended to be merged. It was an experiment to show that the problem was indeed Solr 6.5.0